scripts/ae2.zs

```diff
@@ -924,11 +929,21 @@ recipes.addShaped("planes_annihilation_fluid", <appliedenergistics2:part:302>, [
 ]);
 
 // ME Terminal
-recipes.addShaped("terminals", <appliedenergistics2:part:380>, [
+recipes.removeByRecipeName("appliedenergistics2:network/parts/terminals");
+recipes.addShaped("terminals_base", <appliedenergistics2:part:380>, [
     [<ore:boltNetherQuartz>, <ore:circuitBasic>, <ore:boltNetherQuartz>],
     [<ore:gtceWireCutters>, <ore:itemIlluminatedPanel>, <ore:gtceScrewdrivers>],
     [<ore:circuitBasic>, <ore:plateDoubleSteel>, <ore:circuitBasic>]
 ]);
+assembler.recipeBuilder()
```

Comment:
This should have a space between the crafting and assembler recipes. Additionally, please confirm it does not conflict with anything.